src/google/protobuf/descriptor.ts

```diff
@@ -845,68 +963,22 @@ export interface ServiceOptions {
    */
   deprecated: boolean;
   /** The parser stores options it doesn't recognize here. See above. */
-  uninterpretedOption: UninterpretedOption[];
-}
 
-export interface MethodOptions {
-  /**
-   * Is this method deprecated?
-   * Depending on the target platform, this can emit Deprecated annotations
-   * for the method, or it will be completely ignored; in the very least,
-   * this is a formalization for deprecating methods.
-   */
-  deprecated: boolean;
-  idempotencyLevel: MethodOptions_IdempotencyLevel;
-  /** The parser stores options it doesn't recognize here. See above. */
   uninterpretedOption: UninterpretedOption[];
 }
+export interface MethodOptions {
+  /**
```

Comment:
order of these changed, but it doesn't change the behavior. This is due to the many nested sub types. Looks good in terms of structure.